reactor-core/src/test/java/reactor/core/publisher/MonoRetryPredicateTest.java

```diff
@@ -34,14 +36,14 @@ public void twoRetryNormalSupplier() {
 
 		StepVerifier.create(Mono.fromCallable(i::incrementAndGet)
 		                        .doOnNext(v -> {
-		                        	if(v < 4) {
+			                        if (v < 4) {
```

Comment:
the tests of removed retry operators should either be moved to `xxxRetryWhenTest` or there should be a comment in that test class pointing to `xxxRetryPredicateTest`. also there are `SuppressWarnings("deprecation")` in these tests that are not necessary anymore